src/utilities/graphDataFactory.ts

```diff
@@ -27,7 +27,7 @@ export function graphDataFactory(): GraphDataFactory {
 
       watch(() => subscription.response, response => {
         if (response) {
-          callback(response)
+          callback(toRaw(response))
```

Comment:
This just makes sure we don't pass any reactive data. Removes vue's proxies which makes the data handling a little cleaner and lighter.